### PR TITLE
Xiaomi BLE now supports some binary sensors

### PR DESCRIPTION
**Pull request by Jc2k**
## Proposed change

Updates the docs after https://github.com/home-assistant/core/pull/76635.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/76635
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/xiaomi_ble.markdown [hunk 1]
```diff
@@ -3,6 +3,7 @@ title: Xiaomi BLE
 description: Instructions on how to integrate Xiaomi BLE devices into Home Assistant.
 ha_category:
   - Sensor
+  - Binary Sensor
```
Comment (bdraco) [inline source/_integrations/xiaomi_ble.markdown]:
```suggestion
  - Binary Sensor
  - Sensor
```

File: source/_integrations/xiaomi_ble.markdown [hunk 2]
```diff
@@ -13,6 +14,7 @@ ha_domain: xiaomi_ble
 ha_config_flow: true
 ha_platforms:
   - sensor
+  - binary_sensor
```
Comment (bdraco) [inline source/_integrations/xiaomi_ble.markdown]:
```suggestion
  - binary_sensor
  - sensor
```

**Review by bdraco — DISMISSED** [review]

**Comment by Jc2k** [comment]
Cheers!

File: source/_integrations/xiaomi_ble.markdown [hunk 3]
```diff
@@ -36,6 +38,12 @@ It is possible that we detect your device because it uses the MiBeacon protocol
 - Voltage
 - Battery
 
+It also supports the following classes of binary sensor:
```
Comment (bdraco) [inline source/_integrations/xiaomi_ble.markdown]:
```suggestion
It also supports the following classes of binary sensors:
```

**Review by bdraco — APPROVED** [review]

**Pull request merged**